### PR TITLE
chore(calendar): expose in story book with default story

### PR DESCRIPTION
**Pull request by OleksandrNechai**
[FX-4061]

### Description

Expose Calendar component through Storybook. In this PR we expose only Default story. Extra stories are to come.

### How to test

- check `https://picasso.toptal.net/fx-4061-expose-calendar-with-default-story/?path=/story/components-calendar--calendar`

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- [x] codemod is created and showcased in the changeset
- [x] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-4061]: https://toptal-core.atlassian.net/browse/FX-4061?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by OleksandrNechai** [comment]
@toptal-anvil ping reviewers

File: packages/picasso/src/Calendar/Calendar.tsx [hunk 1]
```diff
@@ -38,20 +38,35 @@ export type CalendarMonthsAmount = 1 | 2
...
   onBlur?: (event: React.FocusEvent<HTMLDivElement>) => void
+  /** Minimum date that can be selected */
   minDate?: Date
+  /** Maximum date that can be selected */
   maxDate?: Date
+  /** Whether to use range mode */
   range?: boolean
+  /** Selected date or date range */
   value?: DateOrDateRangeType
+  /** Active (visible) month of the calendar that required for manual entering of a single date */
```
Comment (sashuk) [inline packages/picasso/src/Calendar/Calendar.tsx:53]:
```suggestion
  /** Active (visible) month of the calendar that is required for manual entering of a single date */
```

File: packages/picasso/src/Calendar/story/index.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { Calendar } from '../Calendar'
+import PicassoBook from '~/.storybook/components/PicassoBook'
+
+const page = PicassoBook.section('Components').createPage(
+  'Calendar',
+  `Container for displaying data in calendar form.
```
Comment (sashuk) [inline packages/picasso/src/Calendar/story/index.jsx:6]: What do you think about leaving a note that more use cases can be seen in the `DatePicker` stories (multi-month, custom rendering, etc.)?

**Review by sashuk — APPROVED** [review]
Left some comments, tested on temploy, looks good!

**Pull request merged**